keypatch.py

```diff
@@ -759,16 +763,16 @@ def __init__(self, kp_asm, address, assembly=None, opts=0):
 
             {FormChangeCb}
             <Endian     :{c_endian}>
-            <Syntax     :{c_syntax}>
-            <Address    :{c_addr}>
-            <Original   :{c_orig_assembly}>
-             <-   Encode:{c_orig_encoding}>
-             <-   Size  :{c_orig_len}>
-            <Assembly   :{c_assembly}>
-             <-   Fixup :{c_raw_assembly}>
-             <-   Encode:{c_encoding}>
-             <-   Size  :{c_encoding_len}>
-            <Padding extra bytes with NOPs:{c_opt_padding}>{c_opt_chk}>
+            <~S~yntax     :{c_syntax}>
+            <A~d~dress    :{c_addr}>
+            <~O~riginal   :{c_orig_assembly}>
+             <-   E~n~code:{c_orig_encoding}>
+             <-   S~i~ze  :{c_orig_len}>
+            <~A~ssembly   :{c_assembly}>
```

Comment:
except the control Assembly & Syntax, other controls (such as Address, Encode, Size, etc) do not accept input. so i think there is no need to setup the keys for them.